src/distributions/uniform.rs

```diff
@@ -96,9 +53,9 @@ pub struct AsciiWordChar;
 impl Distribution<isize> for Uniform {
     fn sample<R: Rng+?Sized>(&self, rng: &mut R) -> isize {
         if mem::size_of::<isize>() == 4 {
-            i32::rand(rng, Uniform) as isize
+            Distribution::<i32>::sample(&Uniform, rng) as isize
         } else {
-            i64::rand(rng, Uniform) as isize
+            Distribution::<i64>::sample(&Uniform, rng) as isize
```

Comment:
Does it make sense to just use `next_u32` and `next_u64` here and for `usize`?

Maybe it is also better to change the conditions to test for 64-bit. See the open issue [Policy for assumptions about the size of `usize`](https://github.com/rust-lang/rfcs/issues/1748). But not a big deal.
```
if mem::size_of::<isize>() == 8 {
    next_u64() as isize
} else {
    // 32-bit or smaller
    next_u32() as isize
}
```